homeassistant/components/elkm1/__init__.py

```diff
@@ -12,6 +12,8 @@
 from homeassistant.helpers.entity import Entity
 from homeassistant.helpers.typing import ConfigType  # noqa
 
+REQUIREMENTS = ['elkm1-lib==0.7.14']
```

Comment:
We don't need this anymore with the manifest.json.

Reply:
ok, removed